Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM ghcr.io/wiiu-env/devkitppc:20230326
 
 ENV DEBIAN_FRONTEND=noninteractive \
  PATH=$DEVKITPPC/bin:$PATH \
- WUT_ROOT=$DEVKITPRO/wut
+ WUT_ROOT=$DEVKITPRO/wut \
+ curl_ver=8.0.1
```

Comment:
Could you change this to CURL_VER, please, just so it's in line with other env variables